### DISCUSSION TITLE
support ssh tunnel as connection endpoint

### DISCUSSION THREAD

**Discussion by andre-merzky** [radical-cybertools/radical.pilot#2412] [Ideas]
This relates to radical-collaboration/hpc-workflows/issues/82: we should provide a tool to set up an ssh tunnel to the target machine, and configure RP/SAGA so that this tunnel is used for all  connections.

It  might make sense to move that tool to RU?

**Comment by iparask** [comment]
I think this is ongoing based on the v2 discussions
